lib/theme_check/language_server/server.rb

```diff
@@ -25,6 +25,15 @@ def initialize(
         @out = out_stream
         @err = err_stream
 
+        # Because programming is fun,
+        #
+        # Ruby on Windows turns \n into \r\n. Which means that \r\n
+        # gets turned into \r\r\n. Which means that the protocol
+        # breaks on windows unless we turn STDOUT into binary mode.
+        #
+        # Hours wasted: 8.
+        @out.binmode
```

Comment:
Do we have some tests for weird unicode chars? I'm worried multibyte chars might not get handled properly if this affects encoding.

Reply:
I think we're covered by the `default_external` setting. 

```ruby
Encoding.default_external = Encoding::UTF_8
```

You make me doubt :P Unit tests + VS Code was working on Windows :P But I didn't try with emojis. I'll add a test :)

Reply:
<img width="843" alt="image" src="https://user-images.githubusercontent.com/4990691/131731474-676bf7f0-4db6-4947-9013-fd954ba457d9.png">

Looks good! Turns out it's pretty hard to test lol. The \r\n is only getting translated on the way out.